pkg/stanza/operator/helper/transformer.go

```diff
@@ -94,10 +96,22 @@ func (t *TransformerOperator) ProcessWith(ctx context.Context, entry *entry.Entr
 
 // HandleEntryError will handle an entry error using the on_error strategy.
 func (t *TransformerOperator) HandleEntryError(ctx context.Context, entry *entry.Entry, err error) error {
+	logFields := []zap.Field{
+		zap.Any("error", err),
+		zap.Any("action", t.OnError),
+	}
+	if entry != nil {
+		toAddFields := []zap.Field{zap.Any("entry.timestamp", entry.Timestamp)}
+		if logFilePath, ok := entry.Attributes[attrs.LogFilePath]; ok {
```

Comment:
Should we just add all the entry's attributes?

Reply:
Good point. It will potentially make log line very long, but shouldn't be a problem and can only help. I'll update the code to log all the entry's attributes.